test/integration/basic_test.go

```diff
@@ -99,6 +99,18 @@ func (m *Minishift) executingOcCommand(command string) error {
 	return nil
 }
 
+func (m *Minishift) executingOcCommandSucceeds(command string) error {
+	err := m.executingOcCommand(command)
+	if err != nil {
+		return err
+	}
+	if lastCommandOutput.ExitCode != 0 && len(lastCommandOutput.StdErr) != 0 {
+		return fmt.Errorf("cmdExit: %d, cmdErr: %s", lastCommandOutput.ExitCode, lastCommandOutput.StdErr)
```

Comment:
Maybe we can also add something "Command did not execute successfully. cmdExit: %d, cmdErr: %s"